File: javascript/easy/2114.mostWordsFound.js
/**
 * @param {string[]} sentences
 * @return {number}
 */
var mostWordsFound = function (sentences) {
    let temp = []
    for (let i = 0; i < sentences.length; i++) {
        temp.push(sentences[i].split(" ").length)
    }

    return Math.max(...temp)
}

console.log(mostWordsFound(["alice and bob love leetcode", "i think so too", "this is great thanks very much"])) // 6
console.log(mostWordsFound(["please wait", "continue to fight", "continue to win"])) // 3

File: python/easy/2114.mostWordsFound.py
from typing import List


class Solution:
    def mostWordsFound(self, sentences: List[str]) -> int:
        temp = []
        for x in sentences:
            temp.append(len(x.split()))

        return max(temp)


if __name__ == "__main__":
    solution = Solution()
    print(
        solution.mostWordsFound(
            [
                "alice and bob love leetcode",
                "i think so too",
                "this is great thanks very much",
            ]
        )
    )  # 6
